resources/create-database-script.sql

```diff
@@ -0,0 +1,24 @@
+create table `Role` (
+	roleID SMALLINT NOT NULL,
+    name varchar(64) NOT NULL,
+    PRIMARY KEY (roleID)
+    );
+
+insert into Role(roleID, name) VALUES (1, 'Admin');
+insert into Role(roleID, name) VALUES (2, 'Employee');
+
+create table `User` (
+	userID SMALLINT NOT NULL,
+    email varchar(64) NOT NULL,
```

Comment:
Email length feels short. Is the a standard for the maximum size for an email address